<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/트레이 아이콘.webp</normaloff>icon/트레이 아이콘.webp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 177, 208);</string>
  </property>
  <widget class="QWidget" name="tagList" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>431</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="TagName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>36</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 36pt &quot;Bahnschrift SemiBold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(150, 177, 208);</string>
   </property>
   <property name="text">
    <string>  Tag</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
